### PR TITLE
Make it more cool :sunglasses: ?

### PR DESCRIPTION
**Pull request by jatindhankhar**
We can use cleaner and concise ES6 to make it more 🆒

### CODE REVIEW

File: jatindhankhar/memory_game_hack.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 classes = ['fa-cube', 'fa-diamond', 'fa-leaf', 'fa-bolt', 'fa-bomb', 'fa-anchor', 'fa fa-paper-plane-o', 'fa fa-bicycle']
 
-for(var i = 0 ; i < classes.length ; i++){
-	var x = document.getElementsByClassName(classes[i]);
-	for(var j=0; j <x.length ; j++){x[j].parentElement.click();}
+for(let targetClass of classes){
+
```
Comment (dev-prakhar) [inline jatindhankhar/memory_game_hack.js]: Extra `\n`?

**Review by dev-prakhar — APPROVED** [review]

File: jatindhankhar/memory_game_hack.js (hunk 1 above)
Comment (jatindhankhar) [reply jatindhankhar/memory_game_hack.js]: Might have missed it. Didn't pass it through a code formatter.

Comment (jatindhankhar) [reply jatindhankhar/memory_game_hack.js]: Fixed.

**Comment by dev-prakhar** [comment]
@jatindhankhar Thanks for the PR 👍

**Pull request merged**